flink-libraries/flink-cep/src/test/java/org/apache/flink/cep/nfa/NFAITCase.java

```diff
@@ -2844,4 +2845,35 @@ public void testSharedBufferClearing() throws Exception {
 			Mockito.verify(accessor, Mockito.times(1)).advanceTime(2);
 		}
 	}
+
+	/**
+	 * Test that can access the value of the previous stage directly in notFollowedBy.
+	 *
```

Comment:
The Java doc could be removed as I think the method name already describes clearly what it does. What do you think?

Reply:
OK